### PR TITLE
public network acess changed

### PR DESCRIPTION
**Pull request by shreejam21**
- changed the variable to false
- and for the examples with firewall and vnet rule to it allows IP ranges provides by the users
- updated the readme to reflect the change

### CODE REVIEW

File: examples/intel-optimized-mssql-server-firewall-rules/main.tf [hunk 1]
```diff
@@ -5,6 +5,7 @@ module "optimized-mssql-server" {
   db_server_name      = "optimized-mssql-server-3"
   db_name             = "optimized-mssql-db-3"
   db_password         = var.db_password
+  db_public_network_access_enabled = true
   tags = { owner = "owner@intel.com",
     duration          = "4"
     sql_public_access = "allow"
```
Comment (kalenarndt) [inline examples/intel-optimized-mssql-server-firewall-rules/main.tf:11]: Map formatting is a bit off for the `tags` input and it needs a `terraform fmt`

File: examples/intel-optimized-mssql-server-virtual-network-rule/main.tf [hunk 2]
```diff
@@ -5,7 +5,8 @@ module "optimized-mssql-server" {
   db_server_name      = "optimized-mssql-server-4"
   db_name             = "optimized-mssql-db-4"
   db_password         = var.db_password
-  tags = { owner = "owner.mistry@intel.com",
+  db_public_network_access_enabled = true
+  tags = { owner = "owner@intel.com",
     duration          = "4"
     sql_public_access = "allow"
   }
```
Comment (kalenarndt) [inline examples/intel-optimized-mssql-server-virtual-network-rule/main.tf:12]: Map formatting is a bit off for the `tags` input and it needs a `terraform fmt`

File: examples/intel-optimized-mssql-server/main.tf [hunk 3]
```diff
@@ -7,7 +7,6 @@ module "optimized-mssql-server" {
   db_password         = var.db_password
   tags = { owner = "owner@intel.com",
     duration          = "3"
-    sql_public_access = "allow"
   }
 
 }
```
Comment (kalenarndt) [inline examples/intel-optimized-mssql-server/main.tf:12]: Map formatting is a bit off for the `tags` input and it needs a `terraform fmt`

**Review by kalenarndt — CHANGES_REQUESTED** [review]
Changes look good. Just small formatting changes in the examples. 

I would also add an issue to swap that `count` for the firewall rules to be a `for_each` since it's predictable

**Comment by shreejam21** [comment]
Added the for_each for firwall_rules
Tested and Formatted
Updated README,md

Can you take a look @kalenarndt

**Review by kalenarndt — APPROVED** [review]
Formatting looks good along with the logic changes from `count` to `for_each`

**Review by lucasmelogithub — APPROVED** [review]
LGMT

**Comment by lucasmelogithub** [comment]
@shreejam21 Can you please validate all examples ran successfully?

**Pull request merged**